downstream/modules/aap-on-azure/proc-azure-nw-direct-peering.adoc

```diff
@@ -0,0 +1,33 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="proc-azure-nw-direct-peering"]
+
+= Configuring Network Peering
+
+You can configure network peering between your Azure network and your VNET in the *Virtual Networks* page of the Azure Portal. 
+
+Within the Azure console, the Azure virtual network is known as _this virtual network_, and the VNET that you want to peer with is known as _remote virtual network_.
+
+In the *Virtual Networks* page in the Azure portal, use the following settings to configure the Azure network and the VNET that you want to peer with the {AAPonAzureNameShort} app:
+
+* Under *This virtual network*, select settings for the {AAPonAzureNameShort} virtual network:
+** *Peering link name*: `hub_to_aap` or another memorable name
```

Comment:
> Peering link name*: `hub_to_aap` or another memorable name

Rather than using the example `hub_to_aap` I think we should just use a `<insert name here>` type formatting, since the rest of the entries aren't using examples.

Reply:
@TarikFD  Yes, I originally omitted the name, but then added it because the two suggested names for "this network" and "remote network"  indicate the direction for the traffic flow.

Do you think it would make sense to add something like Enter <_hub_to_aap_peering_link_name_> ?

Reply:
I see what you mean. In that case I think using `<hub_to_aap_peering_link_name>` works